docs/reference/c++_support.rst

```diff
@@ -0,0 +1,173 @@
+.. meta::
+  :description: This chapter describes the C++ support of the HIP ecosystem
+  ROCm software.
+  :keywords: AMD, ROCm, HIP, C++
+
+*******************************************************************************
+C++ language support
+*******************************************************************************
+
+The ROCm platform enables the power of combined C++ and HIP (Heterogeneous-computing
+Interface for Portability) code. This code is compiled with a ``clang`` or ``clang++``
+compiler. The official compilers support the HIP platform, but for the most up-to-date
+feature set, use the ``amdclang`` or ``amdclang++`` components included in the ROCm
```

Comment:
amdclang and amdclang++ are simple wrapper programs which launches clang or clang++ under the same directory. They are so named to differentiate from clang or clang++ coming with the system default packages.